distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1985,40 +1770,48 @@ public void testCancelDownload() throws Exception {
                 RESOURCE_DOWNLOAD_ENDPOINT_ROOT.getUrl() + "?source=" + CSW_STUB_SOURCE_ID
                         + "&metacard=" + metacardId;
 
-        // @formatter:off
-        String downloadId = given().auth().preemptive().basic(LOCALHOST_USERNAME, LOCALHOST_PASSWORD)
-                .get(startDownloadUrl).then().log().all()
-                .extract().jsonPath().getString("downloadId");
-        // @formatter:on
+        given().auth()
+                .preemptive()
+                .basic(LOCALHOST_USERNAME, LOCALHOST_PASSWORD)
+                .get(startDownloadUrl)
+                .then()
+                .log()
+                .all();
 
-        localhostCometDClient.cancelDownload(downloadId);
+        expect("Waiting for download to start.").within(30, SECONDS)
+                .until(() -> localhostCometDClient.getDownloadIds()
+                        .size() > 0);
 
-        expect("Waiting for notifications.").within(10, SECONDS)
-                .until(() -> localhostCometDClient.getMessages(NOTIFICATIONS_CHANNEL)
-                        .size() == 1);
+        assertThat(localhostCometDClient.getDownloadIds()
```

Comment:
Slowing the download is not actually sufficient. I recognize that you can reduce the likelihood of a false negative by doing so, but you can't ensure no false negatives. Only a true block of some sort - a lock, latch, etc. - could ensure it. If there isn't one, then our choices are to introduce yet another flaky, unreliable test - *but at least document it as such for once* - find a way to put in a true block, or not include the test because of its reliability issues.

@lessarderic and @pklinef do you have any thoughts on this?